src/lifetimes.md

```diff
@@ -47,21 +48,24 @@ likely desugar to the following:
 
 <!-- ignore: desugared code -->
 ```rust,ignore
-// NOTE: `'a: {` and `&'b x` is not valid syntax!
-'a: {
+// NOTE: `'global_lifetime: {` and `&'x_lifetime x` is not valid syntax!
+'global_lifetime: {
     let x: i32 = 0;
-    'b: {
-        // lifetime used is 'b because that's good enough.
-        let y: &'b i32 = &'b x;
-        'c: {
-            // ditto on 'c
-            let z: &'c &'b i32 = &'c y;
+    'x_lifetime: {
+        // lifetime used is 'x_lifetime because that's good enough.
+        let y: &'x_lifetime i32 = &'x_lifetime x;
+        'y_lifetime: {
+            // ditto on 'y_lifetime
+            let z: &'y_lifetime &'x_lifetime i32 = &'y_lifetime y;
         }
     }
 }
 ```
 
-Wow. That's... awful. Let's all take a moment to thank Rust for making this easier.
+Wow. That's... awful. Let's all take a moment to thank Rust for making this
+easier. To make matter worse, traditionally, lifetimes are not as descriptive
+and simply called `'a`, `'b`, and so on. We will respect this tradition going
+forward.
```

Comment:
I don't think this change is much helpful for readers and worth rephrasing so. As stated in the intro, we assume readers have a basic knowledge of Rust, actually, https://doc.rust-lang.org/nightly/book/ch10-03-lifetime-syntax.html#lifetime-annotation-syntax explains it and that should be enough.